com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
```

Comment:
you don't need that. It's unmaintainable.

Reply:
Sorry, I am not sure I fully understand. Do you mean the script file is unmaintainable? Or the only_arches part? Any hints on how to resolve this? It is my first time trying to package for Flathub and while I could use flatpak-builder to locally export to a flatpak, I understand that the automated build process on Flathub may be different and what worked locally for me may not work here.

Reply:
This script is excessiely complex for no reason. It's max 5 lines of build-commands

Reply:
I have deleted the script file.